snow/networking/sender/sender_test.go

```diff
@@ -204,8 +204,17 @@ func TestTimeout(t *testing.T) {
 	bootstrapper.GetAncestorsFailedF = failed
 	bootstrapper.GetFailedF = failed
 	bootstrapper.QueryFailedF = failed
-	bootstrapper.AppRequestFailedF = failed
-	bootstrapper.CrossChainAppRequestFailedF = func(ctx context.Context, chainID ids.ID, _ uint32) error {
+	bootstrapper.AppRequestFailedF = func(ctx context.Context, nodeID ids.NodeID, requestID uint32, _ *common.AppError) error {
+		require.NoError(ctx.Err())
+
+		failedLock.Lock()
+		defer failedLock.Unlock()
+
+		failedVDRs.Add(nodeID)
+		wg.Done()
+		return nil
```

Comment:
is it a cleanup that was needed before common.AppErr or is it related to these changes?

Reply:
This looks like a copy-paste error. I can delete this.

Reply:
We also can't just call `failed` inside of here because the linter flags `failed` for always returning `nil` (not sure why it doesn't notice it if we copy-paste)....